variables.tf

```
@@ -27,4 +27,10 @@ variable "website_service_storage_enabled" {
 }
 
 variable "websites_port" {
+}
+
+variable "log_level" {
```

Similar to the above but as an extra/alternative approach, this variable might be nice to have a default value for:
```suggestion
variable "log_level" {
    default = "INFO"
```

Apart from saving a variable needing to be passed through everywhere, it can be a good place to have a clear expectation as a team what we think the "standard" log level is

---

Thanks Jack - I've made that update now.